test-case/check-playback.sh

```diff
@@ -99,5 +100,6 @@ do
     done
 done
 
+pkill -9 aplay
```

Comment:
If the value of $? is 0, then the previous process did terminate normally. It's overkill.

Reply:
If its overkill, how are we still causing all the failures?